RiotSwiftUI/Modules/Common/Util/SecondaryActionButtonStyle.swift

```diff
@@ -27,25 +27,24 @@ struct SecondaryActionButtonStyle: ButtonStyle {
         configuration.label
             .padding(12.0)
             .frame(maxWidth: .infinity)
-            .foregroundColor(strokeColor(configuration.isPressed))
+            .foregroundColor(customColor ?? theme.colors.accent)
             .font(theme.fonts.body)
             .background(RoundedRectangle(cornerRadius: 8)
                             .strokeBorder()
-                            .foregroundColor(strokeColor(configuration.isPressed)))
-            .opacity(isEnabled ? 1.0 : 0.6)
+                            .foregroundColor(customColor ?? theme.colors.accent))
+            .opacity(opacity(when: configuration.isPressed))
     }
     
-    func strokeColor(_ isPressed: Bool) -> Color {
-        if let customColor = customColor {
-            return customColor
-        }
-        
-        return isPressed ? theme.colors.accent.opacity(0.6) : theme.colors.accent
+    private func opacity(when isPressed: Bool) -> CGFloat {
+        guard isEnabled else { return 0.6 }
```

Comment:
Optional: We can make a constant for the `0.6` even internal in the method.

Reply:
BTW, I didn't ignore this. I'm going to come back to it in the [DesignKit PR](https://github.com/vector-im/element-x-ios/pull/64/files) instead of here :)